code/modules/mob/living/silicon/robot/laws.dm

```diff
@@ -19,17 +14,6 @@
 		who = world
 	else
 		who = src
-	if(lawupdate)
-		if (connected_ai)
-			if(connected_ai.stat || connected_ai.control_disabled)
-				to_chat(src, "<b>AI signal lost, unable to sync laws.</b>")
-
-			else
-				lawsync()
-				to_chat(src, "<b>Laws synced with AI, be sure to note any changes.</b>")
-		else
-			to_chat(src, "<b>No AI selected to sync laws with, disabling lawsync protocol.</b>")
-			lawupdate = FALSE
```

Comment:
Where did this behavior go?

Reply:
The checks are done inside `lawsync` except `connected_ai.control_disabled` which I forgot about. It doesn't make sense to remotely set `lawupdate` when wireless is disabled. It's there because the Show Laws verb manually syncs laws which I don't think it should.